Web/Models/Roadmap.cs

```diff
@@ -8,27 +9,52 @@ namespace Web.Models.Roadmap
 {
     public class Create
     {
-        [Required(ErrorMessage = "İsim zorunlu")]
+        //[Required(ErrorMessage = "İsim zorunlu")]
         public string Name { get; set; }
-        [Range(1, int.MaxValue, ErrorMessage = "1 den küçük değer giremezsin.")]
+        //[Range(1, int.MaxValue, ErrorMessage = "1 den küçük değer giremezsin.")]
         public int Visibility { get; set; } = 1;
-        [DataType(DataType.Date, ErrorMessage = "Geçerli bir tarih girin")]
+        //[DataType(DataType.Date, ErrorMessage = "Geçerli bir tarih girin")]
         public DateTime? StartDate { get; set; }
-        [DataType(DataType.Date, ErrorMessage = "Geçerli bir tarih girin")]
+        //[DataType(DataType.Date, ErrorMessage = "Geçerli bir tarih girin")]
         public DateTime? EndDate { get; set; }
     }
 
+    public class CreateValidator : AbstractValidator<Create>
+    {
+        public CreateValidator()
+        {
+
+            RuleFor(x => x.Name).NotNull().WithMessage("İsim Zorunlu");
+            RuleFor(x => x.Visibility).GreaterThanOrEqualTo(1);
+            RuleFor(x => x.StartDate).NotNull().WithMessage("Geçerli bir tarih girin.");
+            RuleFor(x => x.EndDate).NotNull().WithMessage("Geçerli bir tarih girin.");
+            RuleFor(x => x.StartDate.Value).GreaterThan(x => x.StartDate.Value).WithMessage("Bitiş Tarihi başlangıç tarihinden büyük olmalıdır!").When(x => x.StartDate.HasValue);
```

Comment:
Bu kuralda StartDate'in değeri StartDate'in değerinden büyük olmalıdır mı demek istiyor :) Ben mi yanlış anlıyorum? @azizegul